app/Models/Post.php

```diff
@@ -22,7 +22,7 @@ class Post extends Model
      * Always load a user's own reaction,
      * if they're logged-in.
      */
-    protected $with = ['reaction', 'tags'];
+    protected $with = ['tags'];
```

Comment:
Let's remove tags too (and same deal, just return `[]` for that field in the transformer).